src/uphold-scripts-lint

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env sh
 
-eslint --cache . $@
+eslint --cache --fix ${@:-.}
```

Comment:
@nunofgs, @rplopes: does this change still make sense? The original change was [this](https://github.com/uphold/uphold-scripts/commit/77cd9abef17b38ee09ccb717895e9b3f787e9330#diff-86c3ff76acf1d46297dcb67c7e1b87d5), but since then there's been a change to this line in #22.

Reply:
I don't think we should autofix by default when running lint. There might be some changes that, when applied, might cause you to change the code so-slightly to improve readability. We have lint running in CI which fails on eslint warns / errors (including prettier formatting).